common/app/views/fragments/amp/stylesheets/liveBlog.scala.html

```diff
@@ -139,3 +139,71 @@
     text-align: center;
     padding-bottom: 1.5rem;
 }
+
+/* Toast
+========================================= */
+
+.toast__space-reserver {
```

Comment:
@johnduffell: We'll still have the toast on the AMP version, albeit using the markup required by `amp-live-list`

https://github.com/guardian/frontend/blob/cc8f17b840e06546bbe44e2b649bfc36a5e8bc03/article/app/views/liveblog/liveBlogBodyContentAMP.scala.html#L37